transcriptic/cli.py

```diff
@@ -179,6 +179,68 @@ def cli(ctx, api_root, email, token, organization, config):
             pass
 
 
+@cli.command("exec")
+@click.argument("autoprotocol", type=click.File("r"), default=sys.stdin)
+@click.option(
+    "--api", "-a", help="The api endpoint of your scle test workcell instance."
```

Comment:
nit: hm not really sure of a better name right now, but the api name could probably be better